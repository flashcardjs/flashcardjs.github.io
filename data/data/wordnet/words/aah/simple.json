{"word": "aah", "definitions": [{"pos": "v", "text": "express admiration and pleasure by uttering `ooh' or `aah'", "examples": ["They oohed and aahed when they unwrapped the presents"], "synonyms": ["ooh"], "antonyms": []}]}