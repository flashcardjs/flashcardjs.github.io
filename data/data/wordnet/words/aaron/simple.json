{"word": "aaron", "definitions": [{"pos": "n", "text": "United States professional baseball player who hit more home runs than Babe Ruth (born in 1934)", "examples": [], "synonyms": ["Aaron", "Henry Louis Aaron", "Hank Aaron"], "antonyms": []}, {"pos": "n", "text": "(Old Testament) elder brother of Moses and first high priest of the Israelites; created the golden calf", "examples": [], "synonyms": ["Aaron"], "antonyms": []}]}