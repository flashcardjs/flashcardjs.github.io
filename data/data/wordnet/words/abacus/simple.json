{"word": "abacus", "definitions": [{"pos": "n", "text": "a tablet placed horizontally on top of the capital of a column as an aid in supporting the architrave", "examples": [], "synonyms": [], "antonyms": []}, {"pos": "n", "text": "a calculator that performs arithmetic functions by manually sliding counters on rods or in grooves", "examples": [], "synonyms": [], "antonyms": []}]}