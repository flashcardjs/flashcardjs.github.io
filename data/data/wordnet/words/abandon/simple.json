{"word": "abandon", "definitions": [{"pos": "n", "text": "the trait of lacking restraint or control; reckless freedom from inhibition or worry", "examples": ["she danced with abandon"], "synonyms": ["wantonness", "unconstraint"], "antonyms": []}, {"pos": "n", "text": "a feeling of extreme emotional intensity", "examples": ["the wildness of his anger"], "synonyms": ["wildness"], "antonyms": []}, {"pos": "v", "text": "forsake, leave behind", "examples": ["We abandoned the old car in the empty parking lot"], "synonyms": [], "antonyms": []}, {"pos": "v", "text": "give up with the intent of never claiming again", "examples": ["Abandon your life to God", "She gave up her children to her ex-husband when she moved to Tahiti", "We gave the drowning victim up for dead"], "synonyms": ["give up"], "antonyms": []}, {"pos": "v", "text": "leave behind empty; move out of", "examples": ["You must vacate your office by tonight"], "synonyms": ["vacate", "empty"], "antonyms": []}, {"pos": "v", "text": "stop maintaining or insisting on; of ideas or claims", "examples": ["He abandoned the thought of asking for her hand in marriage", "Both sides have to give up some claims in these negotiations"], "synonyms": ["give up"], "antonyms": []}, {"pos": "v", "text": "leave someone who needs or counts on you; leave in the lurch", "examples": ["The mother deserted her children"], "synonyms": ["forsake", "desolate", "desert"], "antonyms": []}]}