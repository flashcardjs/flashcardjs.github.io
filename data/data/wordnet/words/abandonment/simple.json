{"word": "abandonment", "definitions": [{"pos": "n", "text": "the act of giving something up", "examples": [], "synonyms": ["forsaking", "desertion"], "antonyms": []}, {"pos": "n", "text": "withdrawing support or help despite allegiance or responsibility", "examples": ["his abandonment of his wife and children left them penniless"], "synonyms": ["desertion", "defection"], "antonyms": []}, {"pos": "n", "text": "the voluntary surrender of property (or a right to property) without attempting to reclaim it or give it away", "examples": [], "synonyms": [], "antonyms": []}]}