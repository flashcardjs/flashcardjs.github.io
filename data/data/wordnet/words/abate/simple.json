{"word": "abate", "definitions": [{"pos": "v", "text": "make less active or intense", "examples": [], "synonyms": ["slake", "slack"], "antonyms": []}, {"pos": "v", "text": "become less in amount or intensity", "examples": ["The storm abated", "The rain let up after a few hours"], "synonyms": ["let up", "slack off", "slack", "die away"], "antonyms": []}]}