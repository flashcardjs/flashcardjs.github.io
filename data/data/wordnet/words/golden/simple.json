{"word": "golden", "definitions": [{"pos": "s", "text": "having the deep slightly brownish color of gold", "examples": ["long aureate (or golden) hair", "a gold carpet"], "synonyms": ["aureate", "gilded", "gilt", "gold"], "antonyms": []}, {"pos": "s", "text": "marked by peace and prosperity", "examples": ["a golden era", "the halcyon days of the clipper trade"], "synonyms": ["halcyon", "prosperous"], "antonyms": []}, {"pos": "s", "text": "made from or covered with gold", "examples": ["gold coins", "the gold dome of the Capitol", "the golden calf", "gilded icons"], "synonyms": ["gold", "gilded"], "antonyms": []}, {"pos": "s", "text": "supremely favored", "examples": ["golden lads and girls all must / like chimney sweepers come to dust"], "synonyms": ["fortunate"], "antonyms": []}, {"pos": "s", "text": "suggestive of gold", "examples": ["a golden voice"], "synonyms": [], "antonyms": []}, {"pos": "s", "text": "presaging or likely to bring good luck", "examples": ["a favorable time to ask for a raise", "lucky stars", "a prosperous moment to make a decision"], "synonyms": ["favorable", "favourable", "lucky", "prosperous"], "antonyms": []}]}